engine/src/main/java/org/terasology/rendering/dag/nodes/BloomBlurNode.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
It is a nitpick, in large part just so I'd have something to say in a review ... :D

I checked out the IDE customization in Gradle (`/config/gradle/ide.gradle`) and remembered we do have a copyright template there with a hard set year of 2016. So after we fix that newly generated copyright headers should appear with the right year. In the meantime it is just in case you notice an outdated header go ahead and fix. But trivial importance :-)